client/consensus/babe/src/tests.rs

```diff
@@ -572,23 +566,23 @@ fn can_author_block() {
 // Propose and import a new BABE block on top of the given parent.
 fn propose_and_import_block<Transaction>(
 	parent: &TestHeader,
-	slot_number: Option<SlotNumber>,
+	slot_number: Option<Slot>,
 	proposer_factory: &mut DummyFactory,
 	block_import: &mut BoxBlockImport<TestBlock, Transaction>,
 ) -> sp_core::H256 {
 	let mut proposer = futures::executor::block_on(proposer_factory.init(parent)).unwrap();
 
 	let slot_number = slot_number.unwrap_or_else(|| {
 		let parent_pre_digest = find_pre_digest::<TestBlock>(parent).unwrap();
-		parent_pre_digest.slot_number() + 1
+		Slot(parent_pre_digest.slot().0 + 1)
```

Comment:
```suggestion
		parent_pre_digest.slot() + 1.into()
```

Reply:
Ahh I added the `Add` impl later :see_no_evil: